client/components/App.jsx

```diff
@@ -1,24 +1,24 @@
-import React, {useState, useEffect} from 'react'
-import {getGreeting} from '../apiClient'
+import React, { useState, useEffect } from 'react'
+import { getGreeting } from '../apiClient'
 
 const App = () => {
-
   const [greeting, setGreeting] = useState('')
   const [count, setCount] = useState(0)
 
   useEffect(() => {
-    getGreeting()
-      .then((greeting) => {
-        console.log(greeting)
-        setGreeting(greeting)
-      })
+    /* eslint-disable promise/catch-or-return, promise/always-return */
+    getGreeting().then((greeting) => {
+      console.log(greeting)
+      setGreeting(greeting)
+    })
+    /* eslint-enable */
```

Comment:
Can you say a bit about making an exception for this rule, here?

Reply:
This one is a little too involved to fix in this task, I'll open an issue on teaching-tasks to tidy this up.

Reply:
https://github.com/dev-academy-programme/teaching-tasks/issues/20 I've created this